kubectl-plugin/test/e2e/kubectl_ray_log_test.go

```diff
@@ -0,0 +1,219 @@
+package e2e
+
+import (
+	"os"
+	"os/exec"
+	"path"
+	"strings"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var requiredFileSet = map[string]string{
+	"stdout.log": "Ray runtime started",
+	"raylet.out": "Ray Event initialized for RAYLET",
+}
+
+var _ = Describe("Calling ray plugin `log` command on Ray Cluster", Ordered, func() {
+	It("succeed in retrieving all ray cluster logs", func() {
+		expectedDirPath := "./raycluster-sample"
+		expectedOutputStringFormat := `No output directory specified, creating dir under current directory using resource name\.\nCommand set to retrieve both head and worker node logs\.\nDownloading log for Ray Node raycluster-sample-head-\w+\nDownloading log for Ray Node raycluster-sample-small-group-worker-\w+`
+
+		cmd := exec.Command("kubectl", "ray", "log", "raycluster-sample", "--node-type", "all")
```

Comment:
Should we create a test namespace first?

Reply:
I don't think creating a test namespace would be useful here. It would mean creating another raycluster which I think would be redundant. We aren't making changes to the raycluster-sample either also don't think it would effect other tests.